### ISSUE TITLE
Request: ability to add the plugin’s name in the commit’s message

### ISSUE THREAD

**Issue by Screenfeed** [10up/action-wordpress-plugin-deploy#18]
If the SVN user on the repository owns several plugins, it can be hard to follow with the current message:
>Update to version $VERSION from GitHub

It could be nice to be able to set an optional environment var containing the plugin’s name, that would display:
>$PLUGIN_NAME: update to version $VERSION from GitHub

Alternatively, a fully customable message would also work of course.

**Comment by helen** [comment]
Sure, that makes sense. Right now the emails .org sends are prepended with `$SLUG:` - maybe it makes sense to stick it at the end instead so it's not so redundant? I'm not sure which views people look at most, I use a dedicated user for our plugins rather than my own .org account so I don't really look at its profile/commit history, but I do check for the email alerts.

**Comment by jeffpaul** [comment]
@Screenfeed per @helen's prior comment, would something like the following work for you?
> Update to version $VERSION from GitHub for $PLUGIN_NAME

In tandem with that, where are you viewing these commit messages that would make prepending/appending the plugin name helpful?

**Comment by Screenfeed** [comment]
Hello @jeffpaul.

Sure, it would also work.  
These messages can be viewed from a repository's plugin page > Development tab > [development log](https://plugins.trac.wordpress.org/log/imagify/).

**Comment by jeffpaul** [comment]
@Screenfeed right so if you're browsing to that development log don't you already know the plugin you're looking at?  What's the benefit of the plugin name in the commit message then?

**Comment by Screenfeed** [comment]
@jeffpaul You're right, this may not be the place I noticed the issue at the time, as I didn't mentioned this information in my first message 🙄
However, I found another place: the [profile page](https://profiles.wordpress.org/greglone/). The second entry says "Update to version 4.0.0 from GitHub" but we don't know which plugin has been updated to 4.0.0.

**Comment by richard-muvirimi** [comment]
One way to address this is to use the last git commit message as an option, and or alternatively allow the supply of a custom commit message just like the version. Though not in the documentation this can be resolved at run time.
